icu4c/source/common/ucase_props_data.h

```diff
@@ -958,6 +958,16 @@ static const uint16_t ucase_props_unfold[370]={
 0x1ff2,0
 };
 
+struct UCaseProps {
```

Comment:
I don't like moving this here and including the data file again. I think it would be cleaner to move it into ucase.h and to add a getter function. Would you mind me making a commit into your branch?

Reply:
Sure, go ahead!

Reply:
```
$ git push
ERROR: Authentication error: Authentication required: You must have push access to verify locks
error: failed to push some refs to 'github.com:iainireland/icu.git'
```

Reply:
Oops, sorry. Gave you access to this repository; try again?

Reply:
Still the same error message.

Reply:
If you can't find a setting to allow me pushing commits, then I can send you a patch file.

Reply:
Sorry, I haven't done this before. You might have to accept an invitation. If that doesn't work, a patch file is fine.

Reply:
Aha, yes, there was an invitation :-}
I pushed my commit. Please take a look.

Reply:
Looks good to me. I will pull and squash your changes.